MrTheGood/projectb_android
# Add memory

Changes:
 * Added add memory page
* add memory uses current location by default
* requests runtime location permission
* use date picker to pick a date 
* has input validation
* idk, it's late, i'm tired. Long story short: I DID IT :D

How to test: 
 1. Confirm memories are added to map and list at correct locations (currently requeres reopening that page to refresh it)
 2. Confirm that the date, location etc are set correctly

Screenshots:
Just run it yourself, I'm too tired right now
